packages/design/src/DataTable/DataTable.story.tsx

```diff
@@ -187,6 +221,27 @@ const extraData: DummyDataType[] = [
   },
 ];
 
+const isoDateStringData: DummyDataISOStringType[] = [
+  {
+    name: 'i-test',
+    desc: 'yet another item',
+    amount: -50,
+    createdDate: new Date(1635364176).toISOString(),
+    removedDate: new Date(1635322403).toISOString(),
+    tags: ['test12: test12'],
+    bool: false,
+  },
+  {
+    name: 'j-test',
+    desc: 'and another',
+    amount: -20,
+    createdDate: new Date(1635364173).toISOString(),
+    removedDate: new Date(1635322403).toISOString(),
```

Comment:
Nit: the two removed dates are actually the same, so you lose the ability to test sorting by removed date in storybook